dev/tools/tokens/lib/template.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+abstract class TokenTemplate {
+  TokenTemplate(this.fileName, this.tokens);
```

Comment:
nit: You could make this `const` since you only have `final` fields in your class

Reply:
Indeed. I just updated this to make them const constructors. Thanks.